ddmark/validation.go

```diff
@@ -241,3 +297,53 @@ func parseIntOrUInt(value reflect.Value) (int, bool) {
 
 	return fieldInt, ok
 }
+
+// checkValueExistOrIsValid checks if a given string marker item name value exist in a unmarshalled struct (converted to a map by structValueToMap)
+// it returns 1 if the value is found and -if applicable- the required value is valid, 0 otherwise
+func checkValueExistsOrIsValid(markerItem string, structMap map[string]interface{}, ruleName string) (int, error) {
```

Comment:
Why using an int instead of a bool?
Even if the method is used by incrementing a counter, it feels to me this method should return a boolean instead. 
It's up to the caller to decide to increment a counter, not to this method purpose IMO

Reply:
agreed